### PR TITLE
test: fix e2e test suite

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- Added new dev dependency: [`wait-for-expect`](https://github.com/TheBrainFamily/wait-for-expect)
  - it will be useful when waiting for certain interactions like drag and drop.
- But it seems the timing control is not working well in CI. so keep the `slowMo` option in CI.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/playwright/utils.ts [hunk 1]
```diff
@@ -66,3 +67,9 @@ export function getTimeStrFromDate(d: TZDate) {
 
   return `${hour < 10 ? `0${hour}` : hour}:${minute < 10 ? `0${minute}` : minute}`;
 }
+
+export function waitForSingleElement(locator: Locator) {
```
Comment (lja1018) [inline apps/calendar/playwright/utils.ts:71]: 로케이터가 둘 이상 집히지 않게 조심해야겠네요

File: playwright.config.ts [hunk 2]
```diff
@@ -1,37 +1,39 @@
 import { PlaywrightTestConfig, devices } from '@playwright/test';
 
+const isCI = !!process.env.CI;
```
Comment (lja1018) [inline playwright.config.ts:3]: 👍 
`isCIEnv`는 어떠신가요?

**Review by lja1018 — APPROVED** [review]
리뷰 완료합니다.

File: playwright.config.ts (hunk 2 above)
Comment (adhrinae) [reply playwright.config.ts:3]: 고려 안해본건 아닌데 굳이라는 생각이 들어서요

File: apps/calendar/playwright/utils.ts (hunk 1 above)
Comment (adhrinae) [reply apps/calendar/playwright/utils.ts:71]: 저걸 보통 가이드 엘리먼트같은거때문에 하나만 엘리먼트를 잡도록 해야하는데 그걸 명시적으로 기다리는 API가 없어서 쓰는거긴해요.

**Comment by dotaitch** [comment]
늦었지만 리뷰 완료합니다.

**Pull request merged**